contentcuration/contentcuration/tests/testdata.py

```diff
@@ -159,22 +190,32 @@ def channel():
 
 
 def user():
-    user = cc.User.objects.create(email='user@test.com')
-    user.set_password('password')
-    user.is_active = True
-    user.save()
+    user, is_new = cc.User.objects.get_or_create(email='user@test.com')
+    if is_new:
+        user.set_password('password')
+        user.is_active = True
+        user.save()
     return user
 
 
 def create_temp_file(filebytes, preset='document', ext='pdf', original_filename=None):
-    """
-    Create a file and store it in Django's object db temporarily for tests.
+    """Old name for create_studio_file."""
+    import warnings
+    warnings.warn('Deprecated function; use create_studio_file instead.', DeprecationWarning)
+    return create_studio_file(filebytes, preset='document', ext='pdf', original_filename=None)
```

Comment:
@kollivier Good idea to keep the old `create_temp_file` around for a bit --- in case there are some PRs in the works that depend on this functin.